K8s-Orch-Tools/trigger_jenkins.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python3 -u
+
+########################################################
+# trigger_jenkins.py
+#
+# This script triggers the jenkins job setup-cortx-cluster-solution-input,
+# which is a simple sanity test that is useful to run prior to merging
+# changes.  This job collects logs and artifacts that may be useful in
+# updating your Jira issue.
+#
+# The job runs on your own Kubernetes cluster.  It assumes
+# that it is ready for CORTX deployment, which means that
+# "prereq-deploy-cortx-cloud.sh" has already been run and
+# CORTX is not currently running.
+#
+#
+#
+# Usage:  trigger_jenkins.py <options>
+#
+#  -s <solution.yaml>  The solution.yaml file that describes your cluster.
+#                    This file will be uploaded to the Jenkins job, which
+#                    copies it to /var/tmp/solution.yaml on the first node
+#                    listed.  This file determines which cortx-all package
+#                    will be used.
+#
+#  -t <tag>  This is the git changeset of the seagate/cortx-k8s repo to use.
+#                    This can be a tag (like v0.0.21) or a sha.
+#
+#  Jenkins token
+#  -------------
+#  Your Jenkins token must be provided to run the Jenkins job.  The Jenkins user
+#  and token must be in the format <user>:<token>.  These may be specified via
+#  command line as -u <user>:<token>, or through the environment variable
+#  JENKINS_USER_TOKEN.
+#
+#  Node access
+#  -----------
+#  The Jenkins job accesses the nodes via ssh.  This script assumes that the
+#  username and password are identical for all nodes.  You specify the password
+#  via command line as -p <password>, or throught the environment variable
+#  NODE_PASSWORD.
+#
+#  By default the username is "root", but this can be overridden via the command
+#  line as "--node-user <username>".
+#
+#
+# Script Output
+# =============
+# After the Job has been run, the artifacts created by the job, the
+# console output, and a summary suitable for pasting into a report
+# are created in ./setup-cortx-cluster-solution-input_<build-number>.
+#
+#
+########################################################
+
+
+
+import argparse
+import datetime
+import re
+import os
+import sys
+import time
+
+from yaml import load, Loader
```

Comment:
```suggestion
```
Importing from top level makes it harder to read, IMO. `yaml.load()` is better than `load()`.